docs/plugins/language.md

```diff
@@ -0,0 +1,207 @@
+# Language
+
+The Language plugin helps to make your App multilanguage.
+
+## Usage
+
+In order to use the Language plugin, import it from the Lightning SDK.
+
+```js
+import { Language } from 'wpe-lightning-sdk'
+```
+
+### Loading on boot
+
+You can automatically load and initialize the Language plugin when your App boots, by specifying a `static` method `language()`
+on the `App`-class in `src/App.js`.
+
+The `language`-method should return which language to use initially. This can either be a string or a Promise that resolves a
```

Comment:
which language to use on initialisation.

Reply:
👍